File: demo-java/src/main/java/com/example/demojava/DemoController.java
package com.example.demojava;


import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class DemoController {

    @GetMapping("/")
    String index(Model model) {
        model.addAttribute("hello", "Hello World!");
        return "pages/demo";
    }
}

File: demo/src/main/kotlin/com/example/demo/DemoController.kt
package com.example.demo

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping;

@Controller
class DemoController {

    @GetMapping("/")
    fun index(model: Model): String {
        model.addAttribute("hello", "Hello World!")
        return "pages/demo"
    }
}
